src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2707,4 +2686,17 @@ export default class Aircraft {
     removeConflict(other) {
         delete this.conflicts[other.getCallsign()];
     }
+
+    /**
+     * Aircraft has "weight-on-wheels" (on the ground)
+     * @for AircraftInstanceModel
+     * @method wow
+     */
+    wow() {
+        const error_allowance = 5;
+        const apt = window.airportController.airport_get();
+        const rwy_elev = apt.getRunway(this.rwy_dep || this.rwy_arr).elevation;
+        const apt_elev = apt.position.elevation;
+        return this.altitude - (rwy_elev || apt_elev) < error_allowance;
```

Comment:
I have a few notes here:

- if a function/method returns a boolean, it should be named with an `is` at the beginning.  so in this case the method should be named `isWow`.  This way, you can tell at a glance that a function/method will return a boolean without having to look it up.  Kind of like with ruby where boolean functions should have a `?` at the end.

- variable names should be spelled out (const/let) and should also be camel case.  it makes it clear for a new developer to understand what is going on.  it also makes things easy to read.  in this case its pretty apparent, but I would prefer to see 
```javascript
apt -> airport 
rwy_elevation -> runwayElevation
apt_elev -> airportElevation
```

- this line `apt.getRunway(this.rwy_dep || this.rwy_arr).elevation` and this line `return this.altitude - (rwy_elev || apt_elev) < error_allowance` concern me a little bit.  I'm not a fan of inline `||` like this because it's hard to read.  Ideally there would be a method in the `AirportModel` that could handle the logic of determining if a runway is an arrival or departure.  Then you could just ask for the runway elevation.  But that is for a future refactor.

- always include a space above a return statement.  this is another readability thing.  it just makes it easier to pick out and separates the assignments/logics in a method from the return.

---

I'm going to approve this as-is, because this file will probably get worked over pretty good in the near future and it does fix the glitch.

I hope I'm not being too harsh on these reviews.  If I am, please let me know and I'll dial it back.  I just want to make sure that we maintain good syntax across the project post v3.0.0.  A lot of the things that I do when I code have become habit, but I know that isn't the case for everybody.  I want to encourage not discourage and I know that constantly calling out syntax errors can be frustrating.  So I apologize in advance if it gets annoying.  It's for the greater good, I promise!